tasks/lzards-backup/src/index.ts

```diff
@@ -31,19 +31,22 @@ import {
 } from './errors';
 import { isFulfilledPromise } from './typeGuards';
 import {
-  MakeBackupFileRequestResult,
+  ApiGranule,
+  BackupConfig,
   HandlerEvent,
+  MakeBackupFileRequestResult,
   MessageGranule,
   MessageGranuleFilesObject,
   MessageGranuleFromStepOutput,
-  ApiGranule,
 } from './types';
 
 const log = new Logger({ sender: '@cumulus/lzards-backup' });
 
 const S3_LINK_EXPIRY_SECONDS_DEFAULT = 3600;
 const CREDS_EXPIRY_SECONDS = S3_LINK_EXPIRY_SECONDS_DEFAULT;
 
+const getLzardsProviderOrDefault = (lzardsProvider: string | undefined) => lzardsProvider || process.env.lzards_provider || '';
```

Comment:
Do we have a helper getter for non-required ENVs? No problem if not. Just checking.

Reply:
I don't believe so, no.   

  Is that something we *should* have?   The helper function in `@cumulus/common/env` was mostly to standardize the thrown error/log message IIRC.       `process.env` is standard enough and/or typescript syntax makes things like `process?.env?.some?.value`  trivial, generally.

Reply:
Nah. No real reason to add it now. I was just asking in case we already had it. If that were the case, we should use it 😄